### DISCUSSION TITLE
Display Profile URL/link

### DISCUSSION THREAD

**Discussion by AmethystWizard** [community/community#4445] [Mobile]
Many job applications and such ask for your github profile link and yet this is displayed nowhere. Or at least not where it should obviously be displayed, such as directly in your profile.

**Comment by sfullerbeckman** [comment]
I agree, that would be helpful if it was displayed somewhere more obvious on mobile devices.

**Comment by thanhcsf** [comment]
Hmm… I haven't seen the benefit of showing a full profile URL. Would it be easy to take screenshots for quick sending?  

We have the username on profile and an icon to share the profile, right?

**Reply by AmethystWizard** [reply]
No, you would use the copy text function of the mobile device to copy the link so you could use the paste function and paste the text wherever you want.
